packages/nextjs/src/session/getSession.ts

```diff
@@ -1,3 +1,4 @@
+import { getPlatform } from '@packages/mixpanel/platform';
```

Comment:
you need to add packages/mixpanel to the dependencies of this lib when you import it

Reply:
We have that in package.json